scraper/src/custom_middleware.py

```diff
@@ -59,23 +58,8 @@ def process_response(self, request, response, spider):
         if response.url == request.url + '#':
             response = response.replace(url=request.url)
 
-        for rule in spider._rules:
-            if not spider.strict_redirect:
-                if rule.link_extractor._link_allowed(response):
-                    continue
-
-                if rule.link_extractor._link_allowed(request):
-                    response.replace(url=request.url)
-                    continue
-            else:
-                if rule.link_extractor._link_allowed(response) and rule.link_extractor._link_allowed(request):
-                    continue
-
-            if request.url in spider.start_urls and spider.scrap_start_urls is False:
-                continue
+        return response
 
-            if not (spider.scrap_start_urls and response.url in spider.start_urls):
-                print("\033[94m> Ignored:\033[0m " + response.url)
-                raise IgnoreRequest()
+        # is_rules_compliant
 
-        return response
+        # return response
```

Comment:
Same here

Reply:
Same comment. Way to have a mark of the old behavior